moveit_core/collision_detection/include/moveit/collision_detection/test_collision_common_panda.h

```diff
@@ -304,7 +304,64 @@ TYPED_TEST_P(DistanceCheckPandaTest, DistanceSingle)
   }
 }
 
+template <class CollisionAllocatorType>
+class DistanceFullPandaTest : public CollisionDetectorPandaTest<CollisionAllocatorType>
```

Comment:
In #505 this is extended from `DistanceCheckPandaTest<CollisionAllocatorType>`

Reply:
https://github.com/ros-planning/moveit/pull/2682/files also does so